openmetadata-ui/src/main/resources/ui/src/constants/Login.constants.ts

```diff
@@ -40,13 +41,12 @@ export const LOGIN_SLIDE = [
   },
 ];
 
-export const LOGIN_SLIDER_SETTINGS = {
-  arrows: false,
```

Comment:
as we were having arrows false, but in the current implementation there is no such property! can you please double-check for the same